### ISSUE TITLE
Include GPhotosShim like app in the Apps

### ISSUE THREAD

**Issue by W1zardK1ng** [GrapheneOS/Apps#368]
Please add an app like [GPhotosShim](https://github.com/CaramelFur/GPhotosShim) in the Graphene os Apps store, i know currently the Apps store includes only the apps made by Graphene OS team or google apps which are not easy to download. And i am not suggesting to include someone else's app also.

Considering this will be a very small but useful app if graphene os team can create an app like GPhotosShim and include in the app store it will be very helpful for people like me who dont want to sideload apps from unknown/unmaintained github repos and uses google playstore or Apps appstore for installing applications.

**Comment by W1zardK1ng** [comment]
> A lot of the time people have these apps installed and then try to install Google Photos in another profile and wonder why they can't or vice versa.

I completely agree with your points but due to some users installing this app and trying to install google photos app in the other profile and failing it doesnt mean that we have to let other users (who knows what they are doing) installing like these apps from some unknown git repos. I think giving clear app description that installing this app makes it impossible to install google photos app in current or other profiles should be sufficient.



> I don't think we want to do this right now.

Again i respect your decision on this, maybe could reconsider in the future.

**Comment by MichaelDevon** [comment]
This would preferably be a very small OS level component installed by default rather than a optional addon, to avoid confusion, and make the UX seamless.
The Open With UI could promt to define the user's preferred gallery app when they tap the photo preview. This would avoid breaking existing setups with Google Photos installed.

**Comment by MichaelDevon** [comment]
> > This would preferably be a very small OS level component installed by default rather than a optional addon
> 
> This would block users from ever being able to install Google Photos when desired

No. Please see full context before making a point:

> The Open With UI could promt to define the user's preferred gallery app when they tap the photo preview. This would avoid breaking existing setups with Google Photos installed.

**Comment by MichaelDevon** [comment]
Are you aware of the existence of the link handler dialogue? What prevents the user from specifying Photos as the default Gallery app?

**Comment by MichaelDevon** [comment]
Its not black magic. GPhotosShim simply forwards the intent to the Gallery of your choice, which could be Google Photos as well.
